code-for-canada/project-thundercat
# [Email Reply/Edit] Reply button formatting

# Description

Changed formatting of the reply buttons to more closely match the style shown in the design docs.

NOTE1: IE does not recognize ":focus-within", (but most other browsers support it) so it does not get the outline when one of the radio buttons has focus:
![image](https://user-images.githubusercontent.com/2746350/56150338-b04e0f80-5f7c-11e9-80af-a954d3b25958.png)


NOTE2: There is a minor glitch where there is no email type if you submit a response without making changes. This already exists on master and is not introduced by this PR. See https://trello.com/c/BLrOPCJC/48-bug-fix-reply-not-defaulting

## Type of change

- Code cleanliness or refactor

## Screenshot

### Before:
#### Reply:
![image](https://user-images.githubusercontent.com/2746350/56143447-8db4fa00-5f6e-11e9-82c5-fbb32ee8eaed.png)

#### Reply all:
![image](https://user-images.githubusercontent.com/2746350/56143456-94437180-5f6e-11e9-9256-db231736eedc.png)

#### Forward:
![image](https://user-images.githubusercontent.com/2746350/56143476-9c9bac80-5f6e-11e9-9791-312574cdce92.png)

### After
#### Without focus:
![image](https://user-images.githubusercontent.com/2746350/56150151-2ef67d00-5f7c-11e9-9ce1-21ab5d569d3c.png)

#### Reply, with focus:
![image](https://user-images.githubusercontent.com/2746350/56150163-403f8980-5f7c-11e9-84a2-77bc7054b591.png)

#### Reply all, with focus:
![image](https://user-images.githubusercontent.com/2746350/56150178-4897c480-5f7c-11e9-8312-c6c3840621ba.png)

#### Forward, with focus:
![image](https://user-images.githubusercontent.com/2746350/56150187-52212c80-5f7c-11e9-8219-c5e881dff786.png)

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Start the eMIB test (Start eMIB -> Pre-test -> Start Test)
2.  Switch to the Inbox tab
3.  "Add email response"
4..  Verify that the buttons look as expected and change as expected

# Checklist

Applicable for all code changes.

~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
